crates/pure-stage/src/types.rs

```diff
@@ -222,14 +225,14 @@ impl<T> DerefMut for MpscSender<T> {
     }
 }
 
-#[expect(dead_code)]
+#[allow(dead_code)]
```

Comment:
Why change `expect` to `allow`? The purpose of `expect` is to get notified when an annotation does not make sense anymore and clean-up irrelevant annotations.